test/StringIndices_spec.js

```diff
@@ -0,0 +1,23 @@
+const StringIndices = require('../public/javascripts/StringIndices')
+const expect = require('chai').expect
+
+describe('StringIndices', function() {
+  let stringTest = new StringIndices('Please, help yourself to some')
```

Comment:
this is going to cause "test bleedover" you need to reinitialize this value `beforeEach` spec

```js
describe('StringIndices', function() {
  let stringTest
  beforeEach(function(){
    stringTest = new StringIndices('Please, help yourself to some')
  })
  …
})
```